jsapp/svg-icons/attach.svg

```diff
@@ -1 +0,0 @@
-<svg height="642" viewBox="0 0 642 642" width="642" xmlns="http://www.w3.org/2000/svg"><path d="m189.75 441.3125v-185.9375c0-87.303125 61.11875-153.125 142.1875-153.125s142.1875 65.821875 142.1875 153.125v153.125h-43.75v-153.125c0-63.371875-41.409375-109.375-98.4375-109.375s-98.4375 46.003125-98.4375 109.375v185.9375c0 38.128125 20.3 76.5625 65.625 76.5625s65.625-38.434375 65.625-76.5625v-164.0625c0-13.16875-3.19375-43.75-32.8125-43.75s-32.8125 30.58125-32.8125 43.75v153.125h-43.75v-153.125c0-52.346875 30.778125-87.5 76.5625-87.5s76.5625 35.153125 76.5625 87.5v164.0625c0 69.715625-46.003125 120.3125-109.375 120.3125s-109.375-50.596875-109.375-120.3125z" fill-rule="evenodd"/></svg>
```

Comment:
This icon is deleted but it was the icon I used for "Connect Projects" in the form view tabs. Maybe I missed it but I don't see another icon I can use as a paperclip :P

Reply:
Brought it back, sorry :D